app/lib/actions/middleware/watch_delegated_proxy_sub_tasks.rb

```diff
@@ -0,0 +1,73 @@
+module Actions
+  module Middleware
+    class WatchDelegatedProxySubTasks < ::Dynflow::Middleware
+      class CheckOnProxyActions; end
+      POLL_INTERVAL = 30
```

Comment:
This might be quite aggressive interval, especially with large scale. Once in 5 minutes could be quite good, perhaps exposing this through settings, just in case.